db/schema.rb

```diff
@@ -273,9 +273,10 @@
   create_table "provider_users_providers", force: :cascade do |t|
```

Comment:
I would expect the schema number to have been updated in this PR. Did you run the migration locally?

Reply:
I have run the migrations locally and nothing changed on my branch.

Reply:
Ah right, the number on this migration is lower than the last merged migration. 🚀

Reply:
@tijmenb This PR sat around for a while because it couldn't be merged until after https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1931 and was then rebased, so I think the latest schema timestamp/number has been preferred over the orignal. The schema file does contain the added index though.